### DISCUSSION TITLE
Quartz configuration

### DISCUSSION THREAD

**Discussion by RefaeloGanesh** [quartznet/quartznet#1219] [Q&A] [answered]
Hi,
I'm try to change the "quartz.threadPool.maxConcurency" config key but get exception :
Quartz.SchedulerException: 'ThreadPool type 'Quartz.Simpl.DefaultThreadPool' props could not be configured.'
SchedulerConfigException: Could not parse property 'maxConcurency' into correct data type: No writable property 'maxConcurency' found

I'm use .NET Core 3.1 Console Application
             Quartz 3.3.2

This is my config code :
`            var properties = new NameValueCollection();
            properties["quartz.threadPool.type"] = "Quartz.Simpl.DefaultThreadPool";
            properties["quartz.threadPool.maxConcurency"] = "3";`

what wrong?

**Comment by RefaeloGanesh** [comment, marked as answer]
There is a typo at the site.... missing r :-)

![image](https://user-images.githubusercontent.com/19961125/121704517-8224d580-cadc-11eb-8817-45d9b5b6d37d.png)

**Reply by lahma** [reply]
Thanks for the report, the documentation should be now fixed.
